astroquery/ibe/core.py

```diff
@@ -156,8 +168,10 @@ def query_region_async(
             SQL-like query string. Required if ``coordinates`` is absent.
         mission : str
             The mission to be used (if not the default mission).
+            Note: this kwarg is not actually used here.
```

Comment:
instead of adding  these notes, please remove the irrelevant kwargs

Reply:
Done.